README.md

```diff
@@ -1,24 +1,22 @@
-#amdmeminfo
+# amdmeminfo
 
 
 Get GDDR5 memory information from AMD Radeon GPUs.  Includes memory timing.
 
 ---
 
-###Installation
+### Installation
 
-* Download ZIP or via Github client.
-* Install AMDAPPSDK/OpenCL library (required for OpenCL functions)
-* `sudo apt-get install libpci-dev`
-* Unzip or git clone
+* Download and unzip or git clone: `git clone https://github.com/sling00/amdmeminfo`
```

Comment:
Wrong URL.

Reply:
I think it is best to leave out the URL, leaving, "Download and unzip or git clone."

Reply:
Fixed in #2